src/postprocess/DOS.jl

```diff
@@ -28,9 +28,10 @@ function NOS(ε, basis, orben; smearing=basis.model.smearing, T=basis.model.temp
     # Note the differences to the DOS and LDOS functions: We are not counting states
     # per BZ volume (like in DOS), but absolute number of states. Therefore n_symeqk
     # is used instead of kweigths. The number of states is counted over a temperature
-    # window of width T, where we assume that the number of states remains constant
-    # in the interval [T - T/2, T + T/2] as ( d/dε f_T(εik - ε') )|_{ε' = ε}
-    # with f_T the smearing function. This gives the factor T in front.
+    # window of width T yielding in analogy to the DOS an integral
+    #     ∫_(T-T/2)^(T+T/2) ( d/dε f_τ(εik - ε') )|_{ε' = ε} dτ
```

Comment:
that is eps - T/2 and eps + T/2, right?

Reply:
Even then I'm not completely sure of that derivation, and esp. of the analogy with the DOS (because it's quite different, and we don't normalize wrt the kpoints for instance). The one where we say that we'd like to count the number of states contributing to the integral and just use a smooth window feels the less bad. I'll try and come up with a clearer (and more mathematically informative) derivation later.

Reply:
The feeling is that we should be able to derive an error bound like `exp(-N)` for the convergence of properties, I'll try and make that more precise.

Reply:
No I did not mean eps +/- T/2 , I meant T (as in the T-dependence of the smearing function), but I'm not sure about this either. It's just what I came up with to show the analogy to the DOS in the implementation. Ok ... let's go with the window argument. In that case `-f'((εnk - ε)/T)` would just be a weight, chosen to be large and positive if a lot happens (large occupation fluctuations) and small otherwise.